__integration__/w3c-forward-compat.test.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+import { expect } from 'chai';
+import StyleDictionary from 'style-dictionary';
+import { fs } from 'style-dictionary/fs';
+import Color from 'tinycolor2';
+import { resolve } from '../lib/resolve.js';
+import { buildPath } from './_constants.js';
+import { clearOutput } from '../__tests__/__helpers.js';
+
+describe('integration', () => {
+  afterEach(() => {
+    clearOutput(buildPath);
+  });
+
+  describe('W3C DTCG draft spec forward compatibility', async () => {
+    const sd = new StyleDictionary({
+      tokens: {
+        colors: {
+          $type: 'color', // $type should be inherited by the children
```

Comment:
In this PR, `$type` is not yet supported, nor its inheritance down the token tree.

Therefore you will notice that the custom/css/color transform doesn't apply, and in the snapshot, the color output is still in hex. In a future PR we should adjust this snapshot to be rgb output, and ensure $type and $type inheritance is supported.